todo-mvc-kitchensink/package.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
It's probably time to change these to scoped dependencies (as the ci is now failing)... though it looks like it's complaining about the usage of the Observable shim now.